### PR TITLE
feat(warpscript): add SYMBOLS function

### PR DESCRIPTION
**Pull request by miton18**

### CODE REVIEW

File: warp10/src/test/java/io/warp10/test/tools/ExportFunctions.java [hunk 1]
```diff
@@ -240,7 +240,7 @@ public static void main(String[] args) throws Exception {
...
     staticFunctions.put(FCT_TYPE_CONVERSION, Lists.newArrayList("->PICKLE","PICKLE->","->JSON","JSON->","TOBOOLEAN","TODOUBLE","TOLONG","TOSTRING","TOTIMESTAMP"));
-    staticFunctions.put(FCT_STACK, Lists.newArrayList("AUTHENTICATE","BOOTSTRAP","COUNTTOMARK","CLEAR","CLEARDEFS","CLEARSYMBOLS","CLEARTOMARK","CSTORE","DEF","DEPTH","DEBUGON","DEBUGOFF","DOC","DOCMODE","DROP","DROPN","DUP","DUPN","EXPORT","ELAPSED","TIMINGS","NOTIMINGS","FORGET","LOAD","MARK","NDEBUGON","PICK","ROLL","ROLLD","ROT","RUN","SNAPSHOT","SNAPSHOTALL","SNAPSHOTALLTOMARK","SNAPSHOTTOMARK","STACKATTRIBUTE","STACKTOLIST","STORE","SWAP","TYPEOF"));
+    staticFunctions.put(FCT_STACK, Lists.newArrayList("AUTHENTICATE","BOOTSTRAP","COUNTTOMARK","CLEAR","CLEARDEFS","CLEARSYMBOLS","CLEARTOMARK","CSTORE","DEF","DEPTH","DEBUGON","DEBUGOFF","DOC","DOCMODE","DROP","DROPN","DUP","DUPN","EXPORT","ELAPSED","TIMINGS","NOTIMINGS","FORGET","LOAD","MARK","NDEBUGON","PICK","ROLL","ROLLD","ROT","RUN","SNAPSHOT","SNAPSHOTALL","SNAPSHOTALLTOMARK","SNAPSHOTTOMARK","STACKATTRIBUTE","STACKTOLIST","STORE","SWAP","TYPEOF","SYMBOLS"));
```
Comment (aurrelhebert) [inline warp10/src/test/java/io/warp10/test/tools/ExportFunctions.java]: When adding it here, you could sort it alphabetically

File: warp10/src/main/java/io/warp10/script/functions/SYMBOLS.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+package io.warp10.script.functions;
+
+import io.warp10.script.NamedWarpScriptFunction;
+import io.warp10.script.WarpScriptStackFunction;
+import io.warp10.script.WarpScriptException;
+import io.warp10.script.WarpScriptStack;
+
+/**
+ * Checks if a symbol is defined and pushes true or false on the stack accordingly
```
Comment (randomboolean) [inline warp10/src/main/java/io/warp10/script/functions/SYMBOLS.java]: That is not the description of what the function does.

**Comment by miton18** [comment]
@hbs It come from this thread https://twitter.com/jcvialatte/status/1189472624704966656

The use case is to list all stored variables in the stack

**Comment by hbs** [comment]
This is what the function does, what typical use case is there for it?

**Comment by hbs** [comment]
Thanks for the PR

**Pull request merged**